README.md

```diff
@@ -125,6 +128,42 @@ environment variable to set signing certificate name (e.g Developer ID Applicati
 Assets are saved to `ui/desktop/electron-app/out/make/`.
 DMG packaged desktop UI is available at asset location as `boundary.dmg`.
 
+### Connect Boundary UI to Boundary local instance
+
+This describes how to connect local Boundary UI to your local instance of Boundary.
+
+This assumes you already have up and running a boundary instance in your local environment with a listener `127.0.0.1:9200` (default behaviour). If you do not have it, please [follow Boundary documentation](https://github.com/hashicorp/boundary#build-and-start-boundary-in-dev-mode) for it.
```

Comment:
"a Boundary dev instance"

Reply:
Maybe change the second sentence to be all one link "Learn how to start a local Boundary dev instance"?